include/config.h

```diff
@@ -118,6 +130,14 @@
 // -- SPECIFIC OBJECT SETTINGS --
 // Allow for retries on collecting the remaining blue coins from a blue coin switch
 #define BLUE_COIN_SWITCH_RETRY
+// Fixes shell cancel
+#define SHELL_CANCEL_FIX
```

Comment:
this doesnt need to be a config.h define, or default. its only useful if you know what you're doing and speedrun tricks are kept by default. at least comment it out
```suggestion
// #define SHELL_CANCEL_FIX
```